torch/quantization/fx/_equalize.py

```diff
@@ -698,3 +703,100 @@ def _convert_equalization_ref(model: GraphModule):
     convert_eq_obs(model, modules, weight_eq_obs_dict)
 
     return GraphModule(model, model.graph)
+
+
+###############################################################################
+# Functions for running the equalized model on the Numeric Suite              #
+###############################################################################
+
+def get_layer_sqnr_dict(model_a: nn.Module, model_b: nn.Module, x: Any) -> Dict[str, float]:
```

Comment:
Can `_get_layer_sqnr_dict` and `_get_equalization_qconfig_dict` be helper functions instead and we have a single top level API for selective_equalization with numeric suite which calls them?